src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.SkeletonReferenceCache.cs

```diff
@@ -252,13 +250,9 @@ private struct SkeletonReferenceCache
                     {
                         logger?.Log($"Successfully emitted a skeleton assembly for {compilation.AssemblyName}");
 
-                        var temporaryStorageService = services.GetRequiredService<ITemporaryStorageServiceInternal>();
-                        var storage = temporaryStorageService.CreateTemporaryStreamStorage();
```

Comment:
TempStorage is only useful if you actually hold onto this storage object (so that something can either read from it in the future, or so we can take the info from it (the name/offset/length in the mmf) and send that to someone else so they can read.  That doesn't happen here, so we can remove this entirely.